tests/test_unit/test_load_poses.py

```diff
@@ -205,3 +205,17 @@ def test_fps_and_time_coords(self, fps, expected_fps, expected_time_unit):
                 ds.coords["time"].data,
                 np.arange(ds.dims["time"], dtype=int) / ds.attrs["fps"],
             )
+
+    @pytest.mark.parametrize(
+        "file_name",
+        [
+            "LP_mouse-face_AIND.predictions.csv",
+            "LP_mouse-twoview_AIND.predictions.csv",
+        ],
+    )
+    def test_load_from_lp_file(self, file_name):
+        """Test that loading pose tracks from valid DLC files
+        returns a proper Dataset."""
+        file_path = POSE_DATA.get(file_name)
```

Comment:
I found why this test is failing.
The `POSE_DATA` here is not the same object as the `datasets.POSE_DATA`. Instead it's specifically defined for testing purposes in the `tests/conftest.py` file.

The two LP fles are missing from there.
You can fix the tests by simply adding the two LP file names in the `pytest_configure()` function.

In the future maybe we will change that to automatically fetch all file names.